File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/trials/I1/kotlin/trial_01/solution.kt
fun main() {
    var state = 7L
    var total = 0L
    var largest = 0L
    var evens = 0
    var joined = ""
    for (i in 0..49) {
        state = (state * 48271) % 2147483647
        val term = state % 1000
        total = total + term
        if (term > largest) {
            largest = term
        }
        if (term % 2 == 0L) {
            evens = evens + 1
        }
        if (i < 5) {
            if (i == 0) {
                joined = joined + term
            } else {
                joined = joined + "-" + term
            }
        }
    }
    println("SUM " + total)
    println("MAX " + largest)
    println("EVENS " + evens)
    println("JOINED " + joined)
}

File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/trials/I1/rust/trial_01/solution.rs
fn main () {
    let mut state: i64 = 7;
    let mut k: i64 = 0;
    let mut total: i64 = 0;
    let mut largest: i64 = 0;
    let mut evens: i64 = 0;
    let mut joined: String = String::new();
    while k < 50 {
        state = (state * 48271) % 2147483647;
        let term: i64 = state % 1000;
        total = total + term;
        if term > largest {
            largest = term;
        }
        if term % 2 == 0 {
            evens = evens + 1;
        }
        if k < 5 {
            if k == 0 {
                joined = format!("{}", term);
            } else {
                joined = format!("{}-{}", joined, term);
            }
        }
        k = k + 1;
    }
    println!("SUM {}", total);
    println!("MAX {}", largest);
    println!("EVENS {}", evens);
    println!("JOINED {}", joined);
}
